src/main/java/com/criteo/vips/VipsImageImpl.java

```diff
@@ -147,6 +147,14 @@ public void resize(Dimension dimension, boolean scale) throws VipsException {
 
     private native void resizeNative(int width, int height, boolean scale) throws VipsException;
 
+    public Max1Result max1() throws VipsException {
+        Max1Result r = new Max1Result();
+        max1Native(r);
+        return r;
+    }
```

Comment:
OPT: you can create the Max1Result object in the native method. Thus, you don't need this method but only `max1Native(Max1Result r)`

Reply:
I wanted to do as little as possible in the native method. It seems cleaner to create a Java object in Java code.